storage/src/mmr/verification.rs

```diff
@@ -3,8 +3,12 @@ use crate::mmr::iterator::PeakIterator;
 use commonware_cryptography::{Digest, Hasher as CHasher};
 
 #[derive(Clone, Debug, PartialEq, Eq)]
```

Comment:
I'm not sure whether its recommended to put pragma like this below or above the comment. I've historically done it below but have no basis for that decision.

Reply:
yeah have also been doing below, this was an oversight. changed.